www/package.json

```diff
@@ -4,23 +4,26 @@
   "scripts": {
     "start": "npm run generate-providers && docusaurus start",
     "build": "npm run generate-providers && docusaurus build",
+    "docusaurus": "docusaurus",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",
+    "serve": "docusaurus serve",
```

Comment:
I think we could call this `start:prod` to be more explicit about what the script does, start the website on "production" mode 👍🏽

Reply:
it's what we have in the docusaurus starter. Renaming it would confuse regular docusaurus users but I don't have a strong opinion

Reply:
Also serve is not really recommended for production usage, mostly used to test the website locally in production mode